libsoftwarecontainer/component-test/networkgateway_componenttest.cpp

```diff
@@ -26,10 +26,10 @@ class MockNetworkGateway :
 {
 public:
     MockNetworkGateway() :
-        NetworkGateway(1, // container id
+        NetworkGateway(11, // container id
+                       std::string(BRIDGE_DEVICE_TESTING),
                        std::string(BRIDGE_IP_TESTING), // bridge ip
-                       std::stoi(BRIDGE_NETMASK_BITS_TESTING) // netmask bits
-        )
+                       std::stoi(BRIDGE_NETMASK_BITS_TESTING)) // bridge netmask 
```

Comment:
This comment makes sense given the name of the config I guess, but this reminds me that the naming of this config is off. Shouldn't it be something like "netmask bit length" instead? I mean, it's not the actual bitmask really.